service/app/models/book.rb

```diff
@@ -0,0 +1,20 @@
+class Book < ActiveRecord::Base
+	include ApplicationModel
+
+	belongs_to :publisher, foreign_key: :publisher_id
+	has_many   :book_spines
```

Comment:
__[rubocop]__
* Tab detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Put one space between the method name and the first argument.